doc/source/user_guide/reshaping.rst

```diff
@@ -728,14 +728,14 @@ Suppose we wanted to pivot ``df`` such that the ``col`` values are columns,
 ``row`` values are the index, and the mean of ``val0`` are the values? In
 particular, the resulting DataFrame should look like:
 
-.. note::
-
-   col   col0   col1   col2   col3  col4
-   row
-   row0  0.77  0.605    NaN  0.860  0.65
-   row2  0.13    NaN  0.395  0.500  0.25
-   row3   NaN  0.310    NaN  0.545   NaN
-   row4   NaN  0.100  0.395  0.760  0.24
+.. code-block:: text
```

Comment:
I still dont' think this will format corectly; rather than trying to fix it this way, simply create a DataFrame and show it.

Reply:
I initially thought of doing so. but requested only 'change `note` to the `code-block`'. If you want I can change it to the DataFrame then call it on `.. code-block:: ipython`

Reply:
well, does this actually render the same as a dataframe?

Reply:
I created a example_dataframe variable. Result after rendering looks like this: [link](https://www.imgpaste.net/image/riE84)
If you OK, I can commit my changes

Reply:
> I created a example_dataframe variable. Result after rendering looks like this: [link](https://www.imgpaste.net/image/riE84)
> If you OK, I can commit my changes

*i will write  DataFrame values on original 'reshaping.rst' file, this values only for show render results.

Reply:
@jreback the point of this block in the docs is to show a possible result, and not an actual dataframe. The full section could maybe be rewritten, but that is outside of scope for this PR.